src/Field/RandomNumberGenerator.java

```diff
@@ -0,0 +1,17 @@
+package Field;
```

Comment:
See my comment on your `test/RandomiseMines.java` file about how to test this class